install.ps1

```diff
@@ -0,0 +1,80 @@
+# Installs KoreBuild into a repo, overwriting all previous build scripts
+# This script is downloaded and executed without any of the corresponding scripts in this repo, so don't assume anything in this repo is available!
+
+param(
+    [string]$KoreBuildRepo,
+    [string]$KoreBuildBranch = "anurse/msbuild") # TEMPORARY until this is merged to dev
+
+# Prerequisites
+if(!(Get-Command git -ErrorAction SilentlyContinue)) {
+    throw "KoreBuild requires Git"
+}
+if(!(Test-Path "$(Get-Location)\.git")) {
+    throw "This install script should be run from the root of a Git repo"
+}
+$Changes = git diff --name-only
+if($Changes) {
+    throw "There are unstaged changes in this repo. Reset or commit them before installing KoreBuild"
```

Comment:
a) Hygiene. Don't install KoreBuild as part of a bigger commit
b) Ensures you can easily see what the install script did

I don't really care, we can remove it, it just seemed like a resonable restriction to place on the auto-installer for KoreBuild. If you want to do otherwise, you can always manually install it.